src/zcl_abapgit_repo_online.clas.abap

```diff
@@ -318,4 +322,29 @@ CLASS ZCL_ABAPGIT_REPO_ONLINE IMPLEMENTATION.
     reset_status( ).
 
   ENDMETHOD.
+
+
+  METHOD get_commit_display_url.
+
+    DATA ls_result TYPE match_result.
+    FIELD-SYMBOLS <ls_provider_match> TYPE submatch_result.
+
+    rv_url = me->get_url( ).
+
+    FIND REGEX '^https:\/\/(?:www\.)?(github\.com|bitbucket\.org)\/' IN rv_url RESULTS ls_result.
+    IF sy-subrc <> 0.
+      zcx_abapgit_exception=>raise( |provider not yet supported| ).
+    ENDIF.
+    READ TABLE ls_result-submatches INDEX 1 ASSIGNING <ls_provider_match>.
+    CASE rv_url+<ls_provider_match>-offset(<ls_provider_match>-length).
+      WHEN 'github.com'.
```

Comment:
Can we do github without the `.com` to make it work for enterprise instances?

Reply:
whats the DN pattern and the show-commit path?

Reply:
can we use #3733 for this?